File: apps/src/chat/frontend/js/packets/VerificationCodePacket.js
export default class VerificationCodePacket {

    constructor(code = "") {
        this.code = code;
    }

    toData() {
        return {
            type: "verification_code",
            code: this.code
        };
    }

    static fromData(data) {
        return new VerificationCodePacket(
            data.verification_code
        );
    }

    getCode() {
        return this.code;
    }
}

File: apps/src/chat/backend/packets/verification_code_packet.py
from .packet import Packet

class VerificationCodePacket(Packet):

    def __init__(self, code):
        self._code = code

    def get_type(self):
        return "verification_code"

    def to_data(self):
        return {
            "type": self.get_type(),
            "code": self._code
        }

    @staticmethod
    def from_data(data):
        return VerificationCodePacket(data["code"])

    def get_code(self) -> str:
        return self._code